### PR TITLE
Implement seek for missing codes in wavestream

### PR DESCRIPTION
**Pull request by alnitak**
> 1. Remove this template before submitting
>
> 2. SoLoud project requires extremely liberal licensing, i.e, no attribution is required for binary forms.
>    Patches should come under zlib, cc0, unlicense or wtfpl license, or if you wish to not retain any
>    rights, don't include a license and the rights of your patch are released to the project owner and
>    included under the SoLoud zlib/libpng licensing for anyone to (ab)use.
>
> 3. Code should generally follow the coding conventions in SoLoud, meaning formatting that's basically 
>    the ANSI format in Artistic Style formatter (or just read some of the code and make yours look the
>    same).
>
> 4. Code should generally follow "Orthodox C++" rules, i.e, not "modern c++". This means minimal template
>    use and basically no "new" c++ features.
>
> 5. Platform-specific code (i.e, backends / sinks) are pretty much expected to break the 3. and 4. above.
>
> 6. To show that you understand and accept the rules above, add your name to the AUTHORS file.
>
> TL;DR: make code similar to what's in SoLoud, don't add new licensing requirements, sign the AUTHORS file, thanks.

### CODE REVIEW

**Comment by alnitak** [comment]
Grazie @lakor64
